Development.md

```diff
@@ -0,0 +1,59 @@
+### Branch Structure
+
+```
+Master
+L- Development
+   L- *Feature Branch*
+   L- *Bug fix branch*
+```
+
+Master branch is for finalized, tested, and demo-ready code.
+
+Working branch will be development. Pushes to development require Pull Request and review.
+
+Sub-branches of development will be used to isolate tasks that are being worked on. These are for new features, fixes, or other tasks.
+
+Create feature/fix/task branches to perform your work in. Include your initials so we keep track of who owns branches easily.
+For example, if I'm (Corey) working on an bug fix task, I'll create a branch such as `cts-fix-uploader`.
+
+When work is completed and tested, make a Pull Request to the development branch. This PR requires review from another developer before it is merged.
+
+When code is ready to be locked in for demo, a Pull Request from `development` to `master` will be created and reviewed.
+
+### Doing Development
+- Always have an issue created for any work you are doing. It should contain:
+  - Description of work to be done
+  - Labels to categorize what type of task it is
+- Make sure to include your new task on the project board in the correct column
+
+### Testing
+- When finished with the work on a task, move it into the "Testing" category
+- Coordinate with someone from your team to have them test it
+- When you are testing a task, make a comment on the tasks that says "I'm testing this."
+- Add the "Testing" label to the task
+  - If you run into bugs or issues, list them in your testing comment and notify the developer
+- Go back and forth between tester and developer until all issues fixed
+
```

Comment:
Can we include something in teh PR process where we reference the PR in the issue comments or vice versa, to keep the issues tied to the PRs?

Reply:
@ryekerjh checkout Bug fixing. Will this do for now while we get this update with the full workflow?

Reply:
@vperezma Yeah, that'll work. We will have to work out the kinks along the way anyway.

Reply:
@ryekerjh approve and merge if possible.